src/modules/75-cf/FFCustomMicroFrontendProps.types.ts

```diff
@@ -6,23 +6,38 @@
  */
 
 import type * as ffServices from 'services/cf'
-import type { useGetEnvironment } from 'services/cd-ng'
+import type {
+  useCreateEnvironment,
+  useDeleteEnvironmentV2,
+  useGetEnvironment,
+  useGetEnvironmentListForProject
+} from 'services/cd-ng'
 import type { useConfirmAction } from '@common/hooks'
 import type { useSyncedEnvironment } from '@cf/hooks/useSyncedEnvironment'
 import type RbacOptionsMenuButton from '@rbac/components/RbacOptionsMenuButton/RbacOptionsMenuButton'
 import type routes from '@common/RouteDefinitions'
 import type { ContainerSpinner } from '@common/components/ContainerSpinner/ContainerSpinner'
-import type { NameSchema } from '@common/utils/Validation'
+import type { IdentifierSchema, NameSchema } from '@common/utils/Validation'
+import type { FeatureIdentifier } from 'framework/featureStore/FeatureIdentifier'
+import type { useLicenseStore } from 'framework/LicenseStore/LicenseStoreContext'
 import type { getIdentifierFromName } from '@common/utils/StringUtils'
 import type * as trackingConstants from '@common/constants/TrackingConstants'
 import type useActiveEnvironment from './hooks/useActiveEnvironment'
 import type SectionNoData from './components/NoData/SectionNoData/SectionNoData'
 import type { EnvironmentSDKKeyType } from './utils/CFUtils'
 
 export interface FFCustomMicroFrontendProps {
-  ffServices: typeof ffServices & { useCDGetEnvironment: typeof useGetEnvironment }
+  ffServices: typeof ffServices & {
+    useCDGetEnvironment: typeof useGetEnvironment
+    useGetEnvironmentListForProject: typeof useGetEnvironmentListForProject
```

Comment:
It's worth prefixing these with `CD` to make it super obvious in the MFE that these are coming from outside FF
```suggestion
    useCDGetEnvironmentListForProject: typeof useGetEnvironmentListForProject
```

Reply:
ooh okay, I'll try that now 👍

Reply:
@knagurski that worked! thank you